AGInputControls/Source/FloatingLabelTextField.swift

```diff
@@ -211,8 +229,20 @@ open class FloatingLabelTextField : FormattingTextField {
     }
     
     private func configurePlaceholder() {
-        guard placeholder != nil else { return }
-        placeholderLabel.text = placeholder
+        guard let placeholder else { return }
+        if isRequired, let requiredText {
```

Comment:
perhaps we need another idea here, because for some reason this solution has visual glitches during animation. You can compare it by switching to main branch